src/dbnode/storage/bootstrap/bootstrapper/peers/source.go

```diff
@@ -132,7 +303,7 @@ func (s *peersSource) Read(
 	namespaces bootstrap.Namespaces,
 	cache bootstrap.Cache,
 ) (bootstrap.NamespaceResults, error) {
-	ctx, span, _ := ctx.StartSampledTraceSpan(tracepoint.BootstrapperPeersSourceRead)
+	_, span, _ := s.instrumentation.peersBootstrapperSourceReadStarted(ctx)
```

Comment:
Nit: maybe not worth returning the unused 1st and 3rd result here (for a cleaner call site)?